core/jazz_ui/src/app/secondary-components/service-overview-multienv/service-overview-multienv.component.html

```diff
@@ -140,33 +140,33 @@
                       <!-- <div>(Per Day)</div> -->
                   </div>
               </li>
-             
+
           </ul>
-      </div> 
+      </div>
   </div>
-         
+
 </div>
-      
+
 
 
 <div *ngIf="!ErrEnv" class="service-overview-wrap relative row" >
   <div class='eachBranch col-md-2 col-sm-5' *ngFor='let item of Environments'  (click)='stageClicked(item.logical_id)'>
       <div *ngIf='noSecEnv' class='empty-row'>No Environments Available</div>
-      
-      <div *ngIf='!noSecEnv'>   
-          <div class='stage-title2 stageDisp'>{{item.friendly_name || item.logical_id}} <span class='icon-icon-rightArrow right-arow'></span> </div>
+
+      <div *ngIf='!noSecEnv'>
+          <div class='stage-title2 stageDisp'>{{item.physical_id || item.friendly_name}} <span class='icon-icon-rightArrow right-arow'></span> </div>
```

Comment:
Friendly name should take the precedence. If friendly name is not provided, we can display branch name (physical_id).

Reply:
made this change